cms/utils/page.py

```diff
@@ -92,6 +93,17 @@ def get_page_from_path(site, path, preview=False, draft=False):
         titles = titles.filter(publisher_is_draft=False)
     else:
         titles = titles.filter(published=True, publisher_is_draft=False)
+    if language_code is not None:
```

Comment:
Could you add some comments through these changes to hopefully make it obvious to anyone reading through, what is happening?

We get quite a broad range of django knowledge from contributors, and I think it's fair to say `annotate` isn't basic functionality so it might just make sure everyone can follow what's happening here. 👍

Reply:
Added a brief description. I'm not sure if code comments are the best way to get a crash course in `annotate()` but hopefully the note and link to Django's documentation will be helpful.